src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,64 @@
-export const Pagination = () => {};
+import React from 'react';
+import classNames from 'classnames';
+import { getNumbers } from '../../utils';
+
+interface Props {
+  total: number,
+  perPage: number,
+  currentPage: number,
+  onPageChange: (page: number) => void,
+}
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage = 1,
```

Comment:
```suggestion
  currentPage,
```
prop is not optional - default value is redundant